### PR TITLE
Fixed Typo Issue

### PR DESCRIPTION
**Pull request by jitendra-cedcoss**
Fixed Typo Issue

### Description (*)
Fixed Typo Issue

### Fixed Issues (if relevant)
Fixed Typo Issue

### Contribution checklist (*)
 - [X] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by Karlasa** [comment]
@magento-engcom-team combine 22712 22705 22701

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on combining the pull requests for you

**Comment by magento-engcom-team** [comment]
@jitendra-cedcoss all pull requests have been successfully combined together: 
 - merged https://github.com/magento/magento2/pull/22712: Fixed Typo Issue
 - merged https://github.com/magento/magento2/pull/22705: Fixed Typo Issue
 - merged https://github.com/magento/magento2/pull/22701: Fixed Typo Issue

File: app/code/Magento/Cms/Test/Mftf/Test/AdminAddVariableToWYSIWYGBlockTest.xml [hunk 1]
```diff
@@ -49,7 +49,7 @@
         <waitForLoadingMaskToDisappear stepKey="waitForPageLoad3"/>
         <!--see Insert Variable button disabled-->
         <see selector="{{VariableSection.InsertVariableBtnDisabled}}" userInput="Insert Variable" stepKey="seeInsertWidgetDisabled" />
-        <!--see Cancel button enabed-->
+        <!--see Cancel button enabled-->
         <see selector="{{VariableSection.CancelBtnEnabled}}" userInput="Cancel" stepKey="seeCancelBtnEnabled" />
         <!--see 4 colums-->
```
Comment (Karlasa) [inline app/code/Magento/Cms/Test/Mftf/Test/AdminAddVariableToWYSIWYGBlockTest.xml:54]: You missed typo here

**Review by Karlasa — CHANGES_REQUESTED** [review]

**Comment by magento-engcom-team** [comment]
Hi @jitendra-cedcoss
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/22729 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Pull request closed without merging**